react/features/invite/components/InfoDialogButton.web.js

```diff
@@ -90,19 +92,7 @@ class InfoDialogButton extends Component {
     constructor(props) {
```

Comment:
I see you nicely transformed this oldish class to a newer flow style, me likely! One more thing I always tell to Saul too is that I prefer using the inheritdoc annotation (without any other annotations) every time I don't intend to change the doc of the inherited field/function. Just if you wanna clean that up too.

Reply:
I used inheritdoc. I also rebased onto latest master (and force pushed).